.github/workflows/ci-contracts.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on: 
+  push:
+    paths:
+      - 'contracts/**'
```

Comment:
Another path to add is the root yarn.lock, otherwise we might miss dependency updates that are still valid within semver constraints, but cause breaking changes within our tests.

Reply:
Spoke with @alexroan and he pointed out that there definitely is a dependency here. During our go tests, we run `abigen`, to test our code against those interfaces. So we'll definitely need to run go tests if `contracts/` changes. I removed this exclusion.